op-node/p2p/gossip_test.go

```diff
@@ -101,6 +102,59 @@ func TestVerifyBlockSignature(t *testing.T) {
 	})
 }
 
+func TestVerifyBlockSignatureWithRemoteSigner(t *testing.T) {
+	t.Skipf("Implement mocking of client rpc calls")
```

Comment:
I'm trying to add tests for the remote signer by mocking the response of the rpc call to `eth_signBlockPayload`. Are there any standard methodology in the monorepo for writing such test cases that I can refer to?

Reply:
You'd have to instantiate a small RPC sever, and attach a mock backend to it, to serve the RPC requests.

See `TestBaseServer` in the `rpc` package of op-service how to make a temporary test RPC server.

Reply:
Awesome, I added the mock server to the tests! Thank you :)